### PR TITLE
doc add url in readme

### PR DESCRIPTION
**Pull request by wilsonmar**
Just a small change to doc to see if my git setup works.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # consul-k8s-prometheus-grafana-hashicups-demoapp
-This repo contains application and dashboard definitions for the Consul Layer 7 observability with Kubernetes guide located at learn.hashicorp.com
+This repo contains application and dashboard definitions for the <a target="_blank" href="https://learn.hashicorp.com/tutorials/consul/kubernetes-layer7-observability">Consul Layer 7 observability with Kubernetes guide located at learn.hashicorp.com</a>
```
Comment (im2nguyen) [inline README.md]: let's use markdown for links!

```suggestion
This repo contains application and dashboard definitions for the [Consul Layer 7 observability with Kubernetes guide](https://learn.hashicorp.com/tutorials/consul/kubernetes-layer7-observability) located at learn.hashicorp.com.
```

Comment (wilsonmar) [reply README.md]: OK. The difference is that GitHub translates to links that take people away from a page.
But using it is more consistent. I'll do it your way from here one.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # consul-k8s-prometheus-grafana-hashicups-demoapp
-This repo contains application and dashboard definitions for the Consul Layer 7 observability with Kubernetes guide located at learn.hashicorp.com
+This repo contains application and dashboard definitions for the ![https://learn.hashicorp.com/tutorials/consul/kubernetes-layer7-observability](Consul Layer 7 observability with Kubernetes guide located at learn.hashicorp.com)
```
Comment (im2nguyen) [inline README.md]: no exclamation points -- also the link goes after 😄 

```suggestion
This repo contains application and dashboard definitions for the [Consul Layer 7 observability with Kubernetes guide](https://learn.hashicorp.com/tutorials/consul/kubernetes-layer7-observability) located at learn.hashicorp.com.
```

File: README.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # consul-k8s-prometheus-grafana-hashicups-demoapp
```
Comment (im2nguyen) [inline README.md]:
```suggestion
# Learn Layer 7 Observability with Prometheus, Grafana, and Kubernetes
```

File: README.md [hunk 4]
```diff
@@ -11,4 +11,6 @@ To fully deploy the app run the following scripts in order. Assumes you have a K
 
 `kubectl apply -f app`
 
-To simulate a load on the applicaton, run `kubectl apply -f traffic.yaml`.
+To simulate a load on the applicaton, run:
+
+`kubectl apply -f traffic.yaml`
```
Comment (im2nguyen) [inline README.md]:
````suggestion
```
kubectl apply -f traffic.yaml
```
````

**Review by im2nguyen — APPROVED** [review]
looks great, thanks Wilson!

**Pull request merged**